### DISCUSSION TITLE
ScriptReference/SceneManagement.EditorSceneManager.MarkSceneDirty

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#640] [Notes]
# ScriptReference/SceneManagement.EditorSceneManager.MarkSceneDirty

https://docs.unity3d.com/ScriptReference/SceneManagement.EditorSceneManager.MarkSceneDirty.html

**Comment by andreasIsCoding** [comment]
If you're looking for a method that clears the scene dirty flag, note that there is an internal method in EditorSceneManager that appears to have been added when the prefab system was reworked:
https://github.com/Unity-Technologies/UnityCsReference/blob/6c8a95ff127619e73519662fa497e242b898f9af/Editor/Mono/EditorSceneManager.bindings.cs#L237
As it's internal, you'll need to use reflection to get at it.
